core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -39,15 +42,14 @@
     private static final String RABBITMQ_TEST_MESSAGE = "Hello world";
     private static final int RABBITMQ_PORT = 5672;
     private static final int MONGO_PORT = 27017;
-
+    private static final File CONTENT_FOLDER = new File(System.getProperty("user.home") + "/.tmp-test-container");
```

Comment:
Why don't you use something like `Files.createTempDirectory()`?

Reply:
Fixed, my bad